app/src/main/java/com/quran/labs/androidquran/ui/util/TranslationsSpinnerAdapter.java

```diff
@@ -93,18 +95,34 @@ public View getDropDownView(int position, View convertView, ViewGroup parent) {
       convertView.setTag(new CheckBoxHolder(convertView));
     }
     holder = (CheckBoxHolder) convertView.getTag();
+    if (position == translationNames.length - 1) {
+      holder.checkBox.setButtonDrawable(null);
+      holder.checkBox.setText(R.string.more_translations);
+      holder.checkBox.setOnClickListener(this.textListener);
+    } else {
+      holder.checkBox.setButtonDrawable(R.drawable.abc_btn_check_material);
```

Comment:
Android Studio is complaining that _'The resource @drawable/abc_btn_check_material is marked as private in com.android.support:appcompat'_. Can we change its visibility somehow (instead of setting to `null` or changing its transparency)?